core/data/tests/anonymous_struct_with_rename/output.py

```diff
@@ -0,0 +1,48 @@
+"""
+ Generated by typeshare 1.11.0
+"""
+from __future__ import annotations
+
+from pydantic import BaseModel, ConfigDict, Enum, Field
```

Comment:
This is for all snapshots but pydantic doesn't have an `Enum` import. I believe it has to be:

`from enum import Enum`